### ISSUE TITLE
No favicon (was: Error during web config)

### ISSUE THREAD

**Issue by ColinDabritz** [fish-shell/fish-shell#1043]
Sometimes while running `fish_config` I will get this error:

```1.0.0.127.in-addr.arpa - - [16/Oct/2013 09:09:52] code 404, message File not found```

A whole interaction might look like this:

```
[my machine details] ~> fish_config
Web config started at 'http://localhost:8002/'. Hit enter to stop.
1.0.0.127.in-addr.arpa - - [16/Oct/2013 09:10:05] code 404, message File not found
```

Shutting down.

It appears to come up shortly after launching.
I can't get it to happen consistently, it's across addresses (:8000, :8001 etc).

The error appears to be indicating a missing file. It's odd to me that it would not find it, but only some of the time.

Any thoughts on refining the steps to reproduce?

**Comment by faho** [comment]
Closed by #2048, no?

**Comment by zanchey** [comment]
Indeed!
